exercises/nucleotide-count/package.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
I updated and customized the package.json

Reply:
`package.json` is kept in sync with the one in the root folder. All exercises have the same `package.json`. If this is different, well lose your customizations. Is there anything special (different from the *main* one) you need?

Reply:
I see, in that case I'll revert it. It just includes some new versions and a customized description and authoring.